draft-ietf-quic-manageability.md

```diff
@@ -790,15 +790,21 @@ signal ({{sec-teardown}}) means that this state must be purged either through
 timers or through least-recently-used eviction, depending on application
 requirements.
 
-{{?RFC4787}} requires a timout that is not less than 2 minutes for most UDP
-traffic.
-However, in pratice, timers are often lower, in the range of 15 to 30 seconds.
-In contrast, {{?RFC5382}} recommends a timeout of more than 2 hours for TCP,
-given that TCP is a connection-oriented protocol with well-defined closure
-semantics. For network devices that are QUIC-aware, it is recommended to also
-use longer timeouts for QUIC traffic, as QUIC is connection-oriented. As such,
-a handshake packet from the server indicates the willingness of the server to
-communicate with the client.
+{{?RFC4787}} requires a timeout that is not less than 2 minutes for most UDP
+traffic.  However, in pratice, timers are often lower, in the range of 15 to 30
+seconds. In contrast, {{?RFC5382}} recommends a timeout of more than 2 hours for
+TCP, given that TCP is a connection-oriented protocol with well-defined closure
+semantics.
+
+As QUIC has features to improve robustness to NAT rebinding, and has no
+observable end-of-connection semantics, devices might choose to further reduce
+the UDP timeout to avoid unnecessary state. However, this has two drawbacks.
+First, not all servers inhabit routing architectures where connections will
+survive a port or address change. Second, an intermediary might require one
+endpoint to re-instantiate the connection for security reasons, further
+reducing the likelihood of successful revival. For these reasons, devices that
```

Comment:
I don't think I understand this second point about re-instantiating the connection for security reason.

Reply:
This is the thing where a server packet does no good for rebinding because the firewall expects the first packet from the client. Can you suggest alternate text?

Reply:
Suggestion below.